src/sql/workbench/browser/designer/designerIssuesTabPanelView.ts

```diff
@@ -13,8 +13,10 @@ import { IListRenderer, IListVirtualDelegate } from 'vs/base/browser/ui/list/lis
 import { localize } from 'vs/nls';
 import { IColorTheme, ICssStyleCollector, IThemeService, registerThemingParticipant } from 'vs/platform/theme/common/themeService';
 import { attachListStyler } from 'vs/platform/theme/common/styler';
-import { problemsErrorIconForeground, problemsInfoIconForeground, problemsWarningIconForeground } from 'vs/platform/theme/common/colorRegistry';
+import { problemsErrorIconForeground, problemsInfoIconForeground, problemsWarningIconForeground, textLinkForeground } from 'vs/platform/theme/common/colorRegistry';
```

Comment:
`'textLinkForeground' is declared but its value is never read.` This error's failing the build.